### DISCUSSION TITLE
We can´t see the entities on the map

### DISCUSSION THREAD

**Discussion by Wackiestfaun51** [kibook/spooner#7] [General]
Hi, I need your help. We did everything according to the procedure. I have data for ymap files. I will paste it into a notepad , save it as ymap file and paste it to the server. But when I go to the map, I don't see anything . Where can the problem be? Thank you for your time.

**Comment by kibook** [comment]
The ymap export for this is very experimental so it's extremely basic and I wouldn't be surprised if there's issues.

When you say you "paste it to the server", what does that mean exactly? With a ymap, you need to stream it by creating a resource for it, putting it in a `stream` folder inside that resource, and add `this_is_a_map "yes"` to `fxmanifest.lua`

**Comment by kibook** [comment]
19b709b made some important updates to the ymap export, fixing how rotation is calculated which might have caused your entities to not appear where you thought they might. Let me know if it fixes the issue you were having.

**Reply by Wackiestfaun51** [reply]
Hi, thank you for your repply. We will try and I will send to you the result. Have a nice day.
